content/en/apps/examples/anc.md

```diff
@@ -1,16 +1,55 @@
 ---
-title: "Antenatal Care Reference App"
-linkTitle: "ANC Reference App"
+title: "Antenatal Care"
+linkTitle: "Antenatal Care"
 weight: 1
 description: >
-  Reference application for CHWs using a mobile app for ANC
-keywords: anc
+  Antenatal care workflows for CHWs using a mobile app
+relatedContent: >
+  apps/concepts
+  apps/reference/forms/app
+  apps/reference/tasks
 ---
 
-Applications built with the CHT are configurable by design, and support an exceptional range of features and health service areas. The ANC Reference App provides you with a template for structuring and organizing their digital health app, it’s configuration code, and its testing framework. It includes a foundation for forms related to pregnancy and delivery, flexible data fields, and analytics, and can be deployed as-is or customized by a developer for your unique context. 
+{{% pageinfo %}}
+This Antenatal Care workflow is a "reference application" that provides a template for structuring and organizing your Community Health Toolkit digital health app, its configuration code and testing framework. It is a great way to learn about the CHT's foundation for forms, data fields, and analytics that can be easily customized to fit your programs context.
```

Comment:
```suggestion
This "reference application" for maternal and newborn health provides a template for structuring and organizing your Community Health Toolkit digital health app, its configuration and test code. It can be used as is, or serve as a great way to learn about the CHT's foundation for forms, data fields, and analytics that can be easily customized to fit your context.
```